src/components/DiffView/styles.module.scss

```diff
@@ -44,6 +44,10 @@
   .diff-line .diff-code-selected {
     background-color: $light-gray;
   }
+
+  .diff-gutter:hover button.btn {
```

Comment:
I think you just needed to wrap the whole thing (including the line number) in `Commentable`. This is working for me:

<details>
<summary>Patch</summary>

```diff
diff --git a/src/components/DiffView/index.tsx b/src/components/DiffView/index.tsx
index 3922d61..f7db532 100644
--- a/src/components/DiffView/index.tsx
+++ b/src/components/DiffView/index.tsx
@@ -295,22 +295,25 @@ export class DiffViewBase extends React.Component<Props> {
     const { isInsert, lineNumber } = change;
     const changeKey = getChangeKey(change);
 
-    let gutter = wrapInAnchor(renderDefault());
+    const defaultGutter = wrapInAnchor(renderDefault());
+    let gutter = defaultGutter;
     if (enableCommenting && isInsert && lineNumber && side === 'new') {
       gutter = (
-        <div className={styles.gutter}>
-          {gutter}
-          <Commentable
-            as="span"
-            id={changeKey}
-            className={styles.commentButton}
-            line={lineNumber}
-            fileName={version.selectedPath}
-            versionId={version.id}
-          >
-            {(addCommentButton) => <>{addCommentButton}</>}
-          </Commentable>
-        </div>
+        <Commentable
+          as="div"
+          id={changeKey}
+          className={styles.gutter}
+          line={lineNumber}
+          fileName={version.selectedPath}
+          versionId={version.id}
+        >
+          {(addCommentButton) => (
+            <>
+              {defaultGutter}
+              <span className={styles.commentButton}>{addCommentButton}</span>
+            </>
+          )}
+        </Commentable>
       );
     }
 
diff --git a/src/components/DiffView/styles.module.scss b/src/components/DiffView/styles.module.scss
index bae2ff5..04a8900 100644
--- a/src/components/DiffView/styles.module.scss
+++ b/src/components/DiffView/styles.module.scss
@@ -44,10 +44,6 @@
   .diff-line .diff-code-selected {
     background-color: $light-gray;
   }
-
-  .diff-gutter:hover button.btn {
-    visibility: visible;
-  }
 }
 
 .extraMessages {
```

</details>

Ideally, we would wrap as much as possible in `Commentable` but I don't know if it's possible to do much more.